lib/runner/cli/clirunner.js

```diff
@@ -233,16 +233,47 @@ CliRunner.prototype = {
       if (Array.isArray(this.argv._source) && (this.argv._source.length > 0)) {
         testsource = this.argv._source;
       } else if (this.argv.group) {
+
         // add support for multiple groups
         if (typeof this.argv.group == 'string') {
           this.argv.group = this.argv.group.split(',');
         }
 
-        if (this.argv.group.length == 1) {
-          testsource = this.findGroupPath(this.argv.group[0]);
+        var groupTestsource = this.findGroupPathMultiple(this.argv.group);
+
+        // for a single src folder, we match the group paths created,
+        // but for multiple src folders, we are less strict on enforcing
+        // the existence of each group in each src folder.  If a group
+        // does not exist in the multiple src folder case, it is removed
+        // from the test path list.
+
+        if (this.settings.src_folders.length === 1) {
+
+          // any incorrect path here will result in a run error
+
+          testsource = groupTestsource;
+
         } else {
-          testsource = this.findGroupPathMultiple(this.argv.group);
+
+          // only when all groups fail to match will there be a run error
+
+          testsource = groupTestsource.filter(fs.existsSync);
```

Comment:
Looks like `/lib/runner/reporter.js` is the only other file to use `existsSync`.  It looks like those cases can be replaced with try..catch on `require`.  The other places where it's used here (`clirunner.js`) would need an exists replacement.  I can go ahead and patch that in now...